icekit/images_api/serializers.py

```diff
@@ -0,0 +1,18 @@
+from django.apps import apps
+
+from rest_framework import serializers
+
+
+Image = apps.get_model('icekit_plugins_image.Image')
```

Comment:
Could you provide a touch of documentation in `docs` and `changelog`? I suggest it doesn't need much more than starting an `topics/apis.rst` doc and copying over the PR comment.